metron-deployment/roles/kibana/README.md

```diff
@@ -1,35 +0,0 @@
-Kibana 4
---------
-
-This role installs Kibana along with the default Metron Dashboard.
-
-### FAQ
-
-#### How do I change Metron's default dashboard?
```

Comment:
That's a really good catch. I'd recommend opening a separate Jira to use the same pickle file that the Ambari deployment uses. Preferably before METRON-745. With a duplicate file, we run the risk of someone like me accidentally desyncing the dashboards. Thoughts?